cfme/tests/test_appliance.py

```diff
@@ -417,6 +417,8 @@ def codename_in_log():
         else:
             return True
 
+    appliance.wait_for_web_ui()
```

Comment:
I can't believe you would make this change :-1: :sob:

Reply:
yeah well I did, so accept it